### ISSUE TITLE
Ability to run on PyTorch model?

### ISSUE THREAD

**Issue by kossnick** [google-research/task_adaptation#2]
I see the input to VTAB is a model hosted at https://www.tensorflow.org/hub. Are there any plans to support PyTorch pre-trained models as input? This benchmarking suite seems extremely well built and designed, and I'd love to take advantage of it with some of our PyTorch models.

**Comment by akolesnikoff** [comment]
At the moment, the task adaptation team does not have the capacity to work on the pytorch variant of the VTAB benchmark. In general, replicating the VTAB in pytorch should not be too hard, as the data input pipeline can be reused and the remaining parts (model loading and the standard SGD optimization loop) should be relatively straightforward to implement.

Unless there will be an open source contribution of the pytorch version of VTAB, we plan to have a look at this issue in the beginning of the next year.

**Comment by sachit-menon** [comment]
Hi, I was wondering if there was any change to the above - either if you're aware of an open-source PyTorch implementation, or if anyone on your team had a chance to look at this sometime this year. I agree with the earlier poster that this seems like a really well-designed benchmark that would be great to see more widely adopted. 

(BTW, congrats on the cool work at ECCV on Big Transfer!)
